nomenclature/definitions/variable/industry_types.yaml

```diff
@@ -0,0 +1,122 @@
+# This list of industries will replace any occurrence of `<industry>`
+# in a variable-definition yaml file
+
+
+#   i.e., The following standard generical structure in economic variables:
+#   CapitalPriceIndex|<industry>:
+#   LaborPriceIndex|<industry>:
+
+<Industry>:
+  iPLNT:
+    description: Crops and vegatable in agri sector
+
+  iANIM:
+    description: Animal production
+
+  iFORE:
+    description: Forestry and logging
+
+  iFISH:
+    description: Fishing and aquaculture
+
+  iFOSM:
+    description: Mining fossil fuels
+
+  iOTHM:
+    description: Mining of metal ores and non-metallic minerals
+
+  iFBTO:
+    description: Manufacturing of food, beverage and tobacco products
+
+  iTXWO:
+    description: Manufacturing of textile, wood and printed products
+
+  iCOKE:
+    description: Manufacturing of coke products
+
+  iREFN:
+    description: Manufacturing of refined petroleum products
+
+  iCHEM:
+    description: Manufacturing of chemicals and chemical products
+
+  iRUBP:
+    description: Manufacturing of rubber and plastic products
+
+  iNMMP:
+    description: Manufacturing of non-metallic mineral products
+
+  iMETP:
+    description: Manufacturing of basic metals and metal products
+
+  iELEC:
+    description: Manufacturing of electronic computer, optical and electrical equipment
+
+  iMACH:
+    description: Manufacturing of machinery and equipment nec and other manufacturing
+
+  iELCO:
+    description: Production of electricity by coal
+
+  iELGA:
+    description: Production of electricity by gas
+
+  iELNU:
+    description: Production of electricity by nuclear
+
+  iELHY:
+    description: Production of electricity by hydro
+
+  iELWI:
+    description: Production of electricity by wind
+
+  iELOI:
+    description: Production of electricity by petroleum and other oil derivatives
+
+  iELBI:
+    description: Production of electricity by biomass and waste
+
+  iELSO:
+    description: Production of electricity by solar photovoltaic or thermal
+
+  iELOC:
+    description: Production of electricity by tide, wave, ocean
+
+  iELGE:
+    description: Production of electricity by Geothermal
+
+  iELNE:
+    description: Production of electricity nec
```

Comment:
What is "nec"?

Reply:
I've just changed the list according to request from @hettieboonman in #65 , and this variable is not considered.

Reply:
Thanks for the clarification!